redcap_bridge/test_redcap/test_project_validation.py

```diff
@@ -9,6 +9,8 @@
                                                   initialize_test_dir)
 
 project_dir = test_directory / 'testfiles' / 'TestProject'
+project_dir_no_validation = test_directory / 'testfiles' / 'TestProject2'
```

Comment:
this is not needed any more, right?

Reply:
and the corresponding TestProject2 configuration files are also not needed any more.

Reply:
Yes I left it just to get your opinion, but I think we can just delete it